packages/theme-patternfly-org/scripts/md/styled-tags.js

```diff
@@ -38,6 +38,8 @@ function styledTags() {
               tr.children
                 .filter(child => child.tagName === 'th')
                 .forEach(th => {
+                  th.properties.className = th.properties.className || '';
+                  th.properties.className += ' pf-m-wrap';
```

Comment:
```suggestion
                  th.properties.classList.add('pf-m-wrap');
```

My JS game is weak, but would something like this work?

Reply:
Nope. This script is converting AST nodes from MD into JSX.

The leading space doesn't matter except for aesthetics so I'm okay with it.